packages/reporter-vue/src/runnables/RunnablesList.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { Runnables } from '../store/reporter-store'
+import { rootRunnable } from '@packages/faker'
+import _ from 'lodash'
+import RunnablesList from './RunnablesList.vue'
+
+describe('runnables list', () => it('renders the runnables list', () => {
```

Comment:
We could try using seed: https://github.com/marak/Faker.js/#setting-a-randomness-seed